packages/cli/src/lib/cmds/help.ts

```diff
@@ -70,6 +71,17 @@ const HELP_MESSAGES = new Map<string, string>(
         '',
         'bubblewrap validate --url=[pwa-url]',
       ].join('\n')],
+      ['install', [
+        'Usage:',
+        '',
+        '',
+        'bubblewrap install',
+        '',
+        '',
+        'Options: ',
+        '--apkFile ................. path to the APK file to be isntalled. Defaults to ' +
```

Comment:
What do you think about adding some option (maybe like `debug` or `verbose`) that prints out the adb command we're executing?

Reply:
done